### ISSUE TITLE
Constantly losing around 115-120 plancks more than intended on each transaction in Privatenet.

### ISSUE THREAD

**Issue by bors2908** [polkadot-js/api#3408]
Hi, guys!

I am having a strange problem with Polkadot privatenet node:
I am constantly losing around 115-120 plancks more than intended on each transaction.

For instance: 

1. Alice balance before transaction: 992517.756588380476
2. I am sending a tx, using polkadot.js api with amount of 0.009 DOT
3. Some time after, I am checking api.rpc.payment.queryInfo(...).partialFee for the fee, which is 0.000171594531
4. Alice balance after sending is 992517.747416785829

992517.747416785829 + 0.000171594531 + 0.009 is 992517.756588380360, which is 116 plancks less, than the balance before the tx. 

Are there some kind of "hidden" fees or burns which I haven't taken into account? Maybe there is a way to calculate them properly? 

Node version 0.8.30 
polkadot.js api 4.3.1

**Comment by jacogr** [comment]
No idea and have honestly not looked at the Rust code since it was originally put in - it has undergone quite a bit of iterations since then. I would have natively assumed it accounts for everything and have not compared myself.

It _could_ be the length fees if it it consistently below. This can be checked with the tx length * the byte fees.

Obviously the fees is an estimation, so heavy blocks may influence subsequent fees, i.e. it is dynamic based on network conditions.

TL:DR Can only guess since I have not looked at the Rust code for accounting.

**Comment by jacogr** [comment]
This issue has been open for 21 days with no activity and is not labelled as an enhancement. It will be closed in 7 days.
